bool is_cancel            #cancel before task, if true will cancel

int8 mode
geometry_msgs/PoseStamped pose

string planner_id         #for global_planner
string controller_id      #for local_planner

int8 NAVI_MODE=1          #constant value for mode variable
int8 TRACK_MODE=2
---
int8 result                #whether or not recept this request
string description


int8 SUCCESS=1
int8 FAILTURE=0
